### DISCUSSION TITLE
emane event listener

### DISCUSSION THREAD

**Discussion by 1-Jone** [coreemu/core#746] [General]
Recently, I I tried to reproduce the demo of [EMANE Emulation Event Log (EEL) Generator]. I prepared eventservice.xml, eelgenerator.xml and scenario.eel. When I ran emaneeventservice eventservice.xml -l 3, I can't see any movement of n1 and n2 in the core-pygui.

**Comment by bharnden** [comment]
You would need to go into `/etc/core/core.conf`

And set to the following values:
```
emane_event_generate = False
emane_event_monitor = True
```

**Comment by 1-Jone** [comment]
Thank you for your answer. I set it up as you said. Everything else is fine. The position change is displayed on the terminal, but the node on the core still does not move.

Is it possible that the nodes on the core can only be moved through the ns2 script? emane cannot move the nodes on the core canvas.

**Comment by bharnden** [comment]
Nope, not just through ns2. There is also a xy to geo translation that occurs, based on the settings in the GUI. If you move things too far or not within scope, then that translation can fail or be invalid values.

CORE itself is not a good viewer for seeing positions move geospatially, even though possible with the proper setup (which is also not trivial).

**Reply by GLtangzero** [reply]
Hello bharnden:
I also test the conditon which 1-Jone has done, I also found the position is changed but not found any change in the gui, is there any other way to let it show the change in gui?
Thanks!

**Reply by bharnden** [reply]
You tested this with the changes I noted?

**Reply by GLtangzero** [reply]
Yes, I changed the 
emane_event_generate = False
emane_event_monitor = True
as you said but only the information of terminal is changed but not show on the GUI.

**Reply by bharnden** [reply]
The CORE GUI is poor for actually trying to see geospatial movements etc.

It is possible whatever you are feeding in results in coordinates outside of scope, based on the mapping. Then nothing would happen as expected.

If you have a hard specific example for all settings and values used, that would help. I dont generally advocate using CORE's GUI for movement, but would use something like https://github.com/USNavalResearchLaboratory/sdt which is supported.

**Reply by GLtangzero** [reply]
Thank you very much for your reply.
I will try to check the tool.
Thanks again!
